proxy-node-gateway/src/main/java/uk/gov/ida/notification/GatewayApplication.java

```diff
@@ -144,8 +141,7 @@ private void registerProviders(Environment environment) {
     }
 
     private void registerExceptionMappers(Environment environment) {
-        environment.jersey().register(new HubResponseExceptionMapper());
-        environment.jersey().register(new AuthnRequestExceptionMapper());
+        environment.jersey().register(new GenericExceptionMapper());
```

Comment:
The two exception mappers the GenericExceptionMapper replaces both log the exception, while GenericExceptionMapper does not. Is this intentional?